pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+markers =
+    online: marks tests that use online resources (deselect with '-m "not online"')
+    slow: marks tests that are slow (deselect with '-m "not slow"')
```

Comment:
Ah I forgot to add one of these during general setup, thanks for putting on in here.